### PR TITLE
Super fast layered maps with Topojson.

### PR DESCRIPTION
**Pull request by nader77**
#118

### CODE REVIEW

**Comment by amitaibu** [comment]
@nader77 is this still relevant / ready for review?

**Comment by nader77** [comment]
@amitaibu This particular post is ready, But it should be a part of a series of posts about `AngularJs` maps, I'll work on them again and i'll let you know.

**Comment by amitaibu** [comment]
I personally don't have much interest in Angular anymore ;) - so maybe change the post a bit so it could a be a standalone, and not part of a series?

**Comment by nader77** [comment]
Yes sure.

**Comment by nader77** [comment]
super_fast_layered_maps_with_topojson

**Comment by nader77** [comment]
@amitaibu 
* I changed the view of the blog post to just talk about the difference between `GeoJSON` and `TopoJSON` and not regard angularJs as an important part.
* I changed my bio. (Tell me if you don't like it, I'll come up with something else - I don't like the old one)
* Fixed the behat test.

Please review.

File: app/_posts/2016-04-07-topojson.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+A map with layers to show you the information upon hovering, clicking, etc..  on a city, province, country, any kind of information you want, Whether you are using [ELM](http://elm-lang.org/), [AngularJs](https://angularjs.org/) or any other library.
+
+<!-- more -->
+ 
+Now, Although here at Gizra we are [moving away from AngularJs]({{ BASE_PATH }}/content/thinking-choosing-elm), We developed this specific map a while ago using AngularJs and I never got around to posting this blog-post but I thought the information here is helpful and should be published. 
+
+So while we were creating this map for [IBLI - Index-Based livestock insurance](http://ibli.ilri.org/category/ibli/), We used [angular-leaflet-directive](https://github.com/tombatossals/angular-leaflet-directive)
+, We got a huge GeoJSON from the client and with the help of this directive we were able to divide the map into provinces and display lots of information on each province.
+ 
+<div class="thumbnail">
```
Comment (amitaibu) [inline app/_posts/2016-04-07-topojson.md]: use the new `{% include thumbnail.html image_path="assets/images/posts/facebook-post/image1.jpg" caption="Amitai Burstein's Facebook Post" %}` format

File: app/_posts/2016-04-07-topojson.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+author: NaderSafadi
+published: true
+---
+
+{% include setup %}
+
+Having an interactive map on your website is quite easy with [Leaflet](http://leafletjs.com/), 
+A map with layers to show you the information upon hovering, clicking, etc..  on a city, province, country, any kind of information you want, Whether you are using [ELM](http://elm-lang.org/), [AngularJs](https://angularjs.org/) or any other library.
+
+<!-- more -->
```
Comment (amitaibu) [inline app/_posts/2016-04-07-topojson.md]: move below image

**Comment by amitaibu** [comment]
Some technical stuff -- https://github.com/Gizra/Gizra#technical-stuff-to-write-posts

**Comment by amitaibu** [comment]
After my comments, please assign to me, I'll review

**Comment by amitaibu** [comment]
@nader77 you missed the [thumb image](https://github.com/Gizra/Gizra#technical-stuff-to-write-posts)

**Comment by nader77** [comment]
@amitaibu My main picture didn't look good as a thumbnail so I didn't add it, I'll try to resize it.

**Comment by amitaibu** [comment]
you can add a different one then.

**Comment by nader77** [comment]
Added a cropped image:
gizra_s_blog

**Comment by nader77** [comment]
@amitaibu Ready for code review.

**Comment by RachelBaram** [comment]
@nader77  Is this still a live PR or can it be closed?

**Comment by nader77** [comment]
@RachelBaram I think it's irrelevant anymore, closing it.

**Pull request closed without merging**